packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -147,11 +148,20 @@ export default class RunPanelStore {
     }
 
     async onRunButtonClick() {
+        let timer_counter = 1;
         if (window.sendRequestsStatistic) {
+            performance.clearMeasures();
             performance.mark('bot-start');
 
-            window.sendRequestsStatistic(false);
-            performance.clearMeasures();
+            this.timer = setInterval(() => {
+                window.sendRequestsStatistic(true);
+                performance.clearMeasures();
+                if (timer_counter === 30) {
+                    clearInterval(this.timer);
+                } else {
+                    timer_counter++;
+                }
+            }, 10000);
```

Comment:
Please add note/comment what is the max time for sending the data while the bot is running, in this case 5mins

Reply:
added